src/dialogs/alwaysonbotDialogs/UpdateProfile/UpdateAddress/getAddressesDialog.ts

```diff
@@ -0,0 +1,294 @@
+import { LuisRecognizer } from 'botbuilder-ai';
+import { Choice, ChoiceFactory, ChoicePrompt, ComponentDialog, DialogTurnResult, ListStyle, PromptValidatorContext, TextPrompt, WaterfallDialog, WaterfallStepContext } from 'botbuilder-dialogs';
+import { CommonPromptValidatorModel } from '../../../../models/commonPromptValidatorModel';
+import { LUISAOSetup } from '../../../../utils/luisAppSetup';
+import { CONTINUE_AND_FEEDBACK_DIALOG_STEP,ContinueAndFeedbackDialog } from '../../Common/continueAndFeedbackDialog';
+import i18n from '../../../locales/i18nconfig';
+import { AddressDetails } from './addressDetails';
+import { CallBackDailog, CALL_BACK_DAILOG_STEP } from './callBackDailog';
+import { ChoiceCheckUpdateAddressDialog, CHOICE_CHECK_UPDATE_ADDRESS_DIALOG } from './choiceCheckUpdateAddressDialog';
+import { UpdateAddressDialog, UPDATE_ADDRESS_DIALOG_STEP } from './updateAddressDialog';
+
+
+const WATERFALL_DIALOG = 'waterfallDialog';
+const CHOICE_PROMPT = 'CHOISE_PROMPT';
```

Comment:
not consistent these two lines. i recommend use the first line implementation.
const CHOICE_PROMPT = 'CHOISE_PROMPT';
const TEXT_PROMPT = 'textPrompt';